spec/importers/customer_importer_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe "Customers importer" do
+  let(:bad) { './spec/fixtures/bad/customers.csv' }
+  let(:good) { './spec/fixtures/good/customers.csv' }
+
+  context "unsuccessful import" do
+    it "raises an error" do
+      expect{CustomersImporter.new(bad).import}
+        .to raise_error(ActiveRecord::RecordInvalid)
+    end
+
+    it "does not create any new records to the database" do
```

Comment:
@ski-climb Would the database clean itself between the first and second tests in this file? If so, would the second test be doing anything?